### DISCUSSION TITLE
Referente ao EF Core

### DISCUSSION THREAD

**Discussion by marcelocaselli** [balta-io/forum#344] [Tira Dúvidas] [answered]
Estou com dúvidas referente a classe Post; nela é criado uma propriedade public int AuthorId assim como também é criado uma propriedade public int CategoryId.........no caso da CategoryId ok, pois existe uma classe Category; referente ao AuthorId não existe uma classe Author. De onde vem essa propriedade AuthorId?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @marcelocaselli, como vai?

Acredito que o Author faz referência ao User, não? Pelo que me lembro, como as características são iguais, um usuário também pode ser um autor, logo não tem necessidade de criar uma tabela separada.

Mas se quiser, pode separar em uma tabela nova. Não tem "certo" ou "errado" aqui, são apenas perspectivas.
